test/test_helper.exs

```diff
@@ -1 +1,7 @@
 ExUnit.start(exclude: [:system])
+
+{:ok, files} = File.ls("./test/helper")
```

Comment:
I wish it could. The `Helper.UDP` requires `Mock`. `Mock` requires `assert` macro. `assert` macro requires use of `ExUnit.Case` and `ExUnit.Case` requires `ExUnit` application to be already started.